### PR TITLE
Add ember-install generator.

### PR DESCRIPTION
**Pull request by abuiles**
Mostly based on https://github.com/emberjs/ember-rails/blob/master/lib/generators/ember/install_generator.rb

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -334,6 +334,27 @@ add the files `ember.js` and `ember.prod.js` to
 `vendor/assets/javascripts`. The file of the same name will override the
 copy in the gem.
 
+If at any point you need to update Ember.js from any of the release channels, you can do that with
```
Comment (abuiles) [inline README.md:337]: Don't feel good copying this from ember-rails ;( but it's explained really well there.

**Comment by bcardarella** [comment]
does ember-rails have tests for it?

**Comment by abuiles** [comment]
@bcardarella they do! I'll add them.

**Comment by rwjblue** [comment]
Yes, there are absolutely tests.  IIRC they use VCR to mock the http requests.

**Comment by abuiles** [comment]
@rjackson @bcardarella tests added.

**Pull request merged**